core/build.gradle

```diff
@@ -12,13 +28,17 @@ repositories {
 }
 
 dependencies {
+    implementation 'org.apache.logging.log4j:log4j-slf4j-impl:2.14.0'
+    implementation 'org.apache.httpcomponents.client5:httpclient5:5.0.3'
+    implementation 'commons-io:commons-io:2.8.0'
     implementation 'com.univocity:univocity-parsers:2.9.0'
     implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.0'
     implementation 'com.google.guava:guava:29.0-jre'
     implementation 'commons-validator:commons-validator:1.6'
     implementation 'com.googlecode.libphonenumber:libphonenumber:8.12.13'
     testImplementation group: 'junit', name: 'junit', version: '4.13'
     testImplementation "com.google.truth:truth:1.0.1"
+    testImplementation 'org.hamcrest:hamcrest:2.2'
```

Comment:
What is hamcrest used for? I don't see another reference to it.

Reply:
It was used for matchers, but should be removed since I ended up not using it. Will remove.